packages/segmented-control/README.md

```diff
@@ -131,8 +131,9 @@ import {
 
 | Prop              | Type                         | Description                                                                                                                                                                 | Default            |
 | ----------------- | ---------------------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ------------------ |
-| `children`        | `React.ReactNode`            | Can be text and/or an icon element                                                                                                                                          |                    |
+| `children`        | `React.ReactNode`            | Can be text.                                                                                                                                                                |                    |
```

Comment:
Small nit - maybe this should say 'should' instead of 'can' to express that's what we're expecting

Reply:
yeah i'll change